src/components/Link/Link.scss

```diff
@@ -18,6 +18,13 @@
   }
 }
 
+.IconLockup {
+  display: inline-flex;
+  align-items: center;
```

Comment:
In order to center the icon with the text, and to prevent the icon from wrapping by itself to the next line, the best solution I found is to wrap the last word and the icon in an inline-flex container.

This way of preventing unwanted wrapping won’t work with languages like Japanese because [those languages don’t use spaces between words](https://en.wikipedia.org/wiki/Japanese_writing_system#Spacing_and_punctuation). I’m not sure what to do about this.